scripts/masterCron.ts

```diff
@@ -37,6 +38,7 @@ const signer = getDefaultSigner("m/44'/60'/0'/0/0", network).connect(provider);
 
 let gasLimits = {
   volOracleCommit: 85000,
+  volOracleAnnualizedVol: 0,
   settleAuction: 0,
```

Comment:
wdyt these gas limits should be

Reply:
50k gas

Reply:
changed gas limits to all lmk if reasonable